README.md

```diff
@@ -163,13 +164,18 @@ The following options can be passed to `start_link/2`:
 * `{min_replicas, non_neg_integer()}`
 
   For each Redis master node, the min number of replicas for the cluster
-  to be considered OK.
+  to be considered OK. Default 1.
 
 * `{close_wait, non_neg_integer()}`
 
   How long to delay the closing of clients that are no longer part of
   the slot map. The delay is needed so that messages sent to the client
-  are not lost in transit.
+  are not lost in transit. Default 10000.
+
+  It's unlikely that you would get any useful response from a node that is no
+  longer part of the slot map, so you may want to set this option to 0 to make
+  pending commands fail as soon as possible and return an error. This option may
+  be deleted in a future version.
```

Comment:
What should we do about this? We need to figure it out.

Reply:
What are the messages in transit mentioned?

An idea: The messages in transit might be commands cast from the ered process, so if the ered_client process exits when such cast is in transit, the command gets no response. It should be enough to wait a few milliseconds though, so the command gets queued up in the ered_client process and then it gets an error reply by `reply_all` call in `ered_client:terminate/3`.

Reply:
It is `ered_cluster` that stops the `ered_client` anyway when it's been figured out that it's no longer part of the slot map, so after some short time, ered stops sending commands to it.

Reply:
Another option is to add some 'stopping' state to `ered_client`, which makes it reject all command immediately, but the process is still alive for some more time to handle messages and perhaps even some response from the socket. Perhaps that's better?